co-log-core/co-log-core.cabal

```diff
@@ -26,8 +26,16 @@ library
                            Colog.Core.Class
                            Colog.Core.Severity
 
-  ghc-options:         -Wall -fhide-source-paths
-  build-depends:       base
+  build-depends:       base >= 4.11 && < 5
+
+  ghc-options:         -Wall
```

Comment:
Could you also bump up version in `co-log-core.cabal` file to `0.0.0.1` and add entry to CHANGELOG? We're not going to release `0.0.0.1` version yet but I would like to show to people who are going to contribute to our projects how to write CHANGELOG.